.gitignore

```diff
@@ -25,4 +25,7 @@ lerna-debug.log*
 
 # typescript
 *.tsbuildinfo
-package-lock.json
\ No newline at end of file
+package-lock.json
+
+#env
+.env
```

Comment:
place local .env inside `/pacgkages/drink-buddy-app/config/env.local` for local hosted database